lib/taskana-core/src/test/java/acceptance/classification/ClassificationQueryAccTest.java

```diff
@@ -1,39 +1,47 @@
 package acceptance.classification;
 
+import static acceptance.DefaultTestEntities.defaultTestClassification;
 import static org.assertj.core.api.Assertions.assertThat;
 
-import acceptance.AbstractAccTest;
 import java.util.List;
+import org.junit.jupiter.api.BeforeAll;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.TestTemplate;
-import org.junit.jupiter.api.extension.ExtendWith;
+import testapi.TaskanaInject;
+import testapi.TaskanaIntegrationTest;
 
 import pro.taskana.classification.api.ClassificationService;
 import pro.taskana.classification.api.models.ClassificationSummary;
-import pro.taskana.common.test.security.JaasExtension;
 import pro.taskana.common.test.security.WithAccessId;
 
 /** Acceptance test for classification queries and authorization. */
-@ExtendWith(JaasExtension.class)
-class ClassificationQueryAccTest extends AbstractAccTest {
+@TaskanaIntegrationTest
+class ClassificationQueryAccTest {
+  @TaskanaInject ClassificationService classificationService;
+
+  @WithAccessId(user = "admin")
```

Comment:
please use `businessadmin` instead.

We try to prevent the usage of `admin` as often as possible ^^

Reply:
corrected :)